platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -204,19 +204,32 @@ void UIThreadInit(@NonNull final GLViewHolder viewHolder, @Nullable final HttpHa
      * Responsible to dispose internals of MapController during Map teardown.
      * If client code extends MapController and overrides this method, then it must call super.dispose()
      */
-    protected synchronized void dispose() {
-
+    protected void dispose() {
```

Comment:
Why take the synchronize call out here? Could you add some information to the code for that, just that someone (me) doesn't go and add it back in!

Reply:
I see your point, but I'm not sure what to document because this `synchronized` wasn't actually doing anything. Before the earlier JNI refactor `MapController` was the object that native methods synchronized on, but now nothing else synchronizes on `MapController`.